lib/index.ts

```diff
@@ -78,17 +78,17 @@ export function jwtDecode(token: string, options?: JwtDecodeOptions) {
   let decoded: string;
   try {
     decoded = base64UrlDecode(part);
-  } catch (e: any) {
+  } catch (e) {
     throw new InvalidTokenError(
-      `Invalid token specified: invalid base64 for part #${pos + 1} (${e.message})`,
+      `Invalid token specified: invalid base64 for part #${pos + 1} (${(e as Error).message})`,
     );
   }
 
   try {
-    return JSON.parse(decoded);
-  } catch (e: any) {
+    return JSON.parse(decoded) as object;
```

Comment:
Should we use `as T`?

Reply:
I assume we can add it to the signature? Would that work ?

Reply:
I've added a generic with both both `JwtHeader` and `JwtPayload` as a possible type, this seems to allow casting this type to `T`, and it also does not end up in the generated type definition. I think this could be a possible solution.

Reply:
Sounds good. What also works would be just <T> without any default value. But I think it doesnt change anything.